src/main/frontend/handler/editor.cljs

```diff
@@ -3097,7 +3097,7 @@
         (state/selection?)
         (select-up-down direction)
 
-        :else
+        (not (state/get-edit-input-id))
```

Comment:
Perhaps a brief comment summarizing why there's a condition here would be helpful as I wouldn't know if I hadn't reviewed